src/js/components/widget.jsx

```diff
@@ -118,7 +118,7 @@ export class WidgetComponent extends Component {
         return <div>
                    <div id='sk-container'
                         className={ classNames.join(' ') }
-                        onTouchStart={ this.onTouchStart }
+                        onTouchStart={ this.onTouchStart.bind(this) }
```

Comment:
was it throwing errors? because `this` in the console is tricky because of transpilation... sometimes in the console it's `_this`